packages/core-app-api/src/app/AppContext.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('v1 consumer', () => {
     };
 
     const renderedHook = renderHook(() => useMockAppV1(), {
-      wrapper: ({ children }) => (
+      wrapper: ({ children }: React.PropsWithChildren<{}>) => (
```

Comment:
It's unfortunate to put upon us as consumers to add this type, should have been added on the renderHook side instead :) Does switching over to the replacement library let us sidestep this? :) https://github.com/testing-library/react-hooks-testing-library#a-note-about-react-18-support